tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -152,12 +151,12 @@
                 Style="{StaticResource CardBodyStackPanelTextBlockStyle}"
                 Text="Package Caches"
                 IsTextSelectionEnabled="True"/>
-            <TextBlock
-                Margin="0 10 0 0"
-                Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}"
-                x:Uid ="PackageCachesDescription"
-                IsTextSelectionEnabled="True"
-                TextWrapping="WrapWholeWords"/>
+            <TextBlock Margin="0 10 0 0" Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" TextWrapping="WrapWholeWords">
+                <Run x:Uid="PackageCachesDescription"/>
+                <Hyperlink NavigateUri="https://learn.microsoft.com/en-us/windows/dev-drive/#storing-package-cache-on-dev-drive">
```

Comment:
see: https://github.com/microsoft/devhome/blob/8a35d2f9d6f30164268afa0760f514dd01e32802/tools/SetupFlow/DevHome.SetupFlow/Views/DevDriveView.xaml#L59

I'd work with @btueffers to get a go.microsoft.com link so if that link changes we don't have to update Dev Home